modules/cms/classes/Page.php

```diff
@@ -88,8 +89,24 @@ public function getLayoutOptions()
         $layouts = Layout::listInTheme($theme, true);
         $result = [];
         $result[null] = Lang::get('cms::lang.page.no_layout');
+        $ignoreNames = FileDefinitions::get('ignoreFiles');
+
         foreach ($layouts as $layout) {
             $baseName = $layout->getBaseFileName();
+
```

Comment:
Please update this logic to mirror the logic used in the [Cms\Classes\MediaLibrary](https://github.com/octobercms/october/blob/master/modules/cms/classes/MediaLibrary.php#L515) class. We like to keep things simple in October :)